Packs/SecurityScorecard/Integrations/SecurityScorecard/SecurityScorecard.py

```diff
@@ -0,0 +1,1203 @@
+            error_message: str = f'{error_response_json.get("message")} ({error_response_json.get("statusCode")})'
+            demisto.error(error_message)
+            raise DemistoException(error_message)
+        except ValueError:
+            raise DemistoException(f'Error parsing response as JSON. Response: {response.status_code} {str(response.content)}')
+
+
+""" HELPER FUNCTIONS """
+
+
+def get_last_run(last_run: str, first_fetch: str):
+
+    """
+    Helper function to return the last incident fetch runtime as
+    a UNIX timestamp.
+
+    Args:
+        ``last_run`` (``str``): The last runtime, ``demisto.getLastRun().get("last_run")``
+        ``first_fetch`` (``str``): The first fetch configuration from ``demisto.params().get("first_fetch"")``
+
+    Returns:
+        ``int``, Last run timestamp.
+    """
+
+    # Check for existence of last run
+    # When integration runs for the first time, it will not exist
+    # Set 3 days by default if the first fetch parameter is not set
+
+    if last_run:
+        return int(last_run)
+    else:
+
+        demisto.debug(f"First fetch is defined as '{first_fetch}'")
+        days_ago = first_fetch
+
+        fetch_days_ago = arg_to_datetime(arg=days_ago, arg_name="first_fetch", required=False)
+
+        demisto.debug(f"getLastRun is 'None' in Integration context, using parameter 'first_fetch' value '{fetch_days_ago}'")
+        demisto.debug(f"{days_ago} => {fetch_days_ago}")
+
+        return int(fetch_days_ago.timestamp())  # type: ignore
+
+
+def incidents_to_import(
+    alerts: List[Dict[str, Any]],
+    last_run: Optional[str] = demisto.getLastRun().get("last_run"),
+    first_fetch: Optional[str] = demisto.params().get("first_fetch", "2 days")
+) -> List[Dict[str, Any]]:
+    """
+    Helper function to filter events that need to be imported.
+    It filters the events based on the `created_at` timestamp.
+    Function will only be called if the SecurityScorecard API returns more than one alert.
+
+    Args:
+        ``alerts``(``List[Dict[str, Any]]``): A list of alerts to sort through.
+        ``last_run``(``str``): UNIX timestamp when the fetch incidents had last run.
+        ``first_fetch`` (``str``): Configured 'First fetch' parameter
+    Returns:
+        ``List[Dict[str, Any]]``: Alerts to import
+    """
+
+    last_run = get_last_run(last_run=last_run, first_fetch=first_fetch)  # type: ignore
+
+    demisto.debug(f"Last run timestamp: {last_run}")
+
+    incidents_to_import: List[Dict[str, Any]] = []
+
+    # Check if there are more than 0 alerts
+    if alerts:
+
+        # The alerts are sorted by ascending date so last alert is the most recent
+        most_recent_alert = alerts[-1]
+
+        most_recent_alert_created_date = most_recent_alert.get("created_at")
+
+        most_recent_alert_timestamp = \
+            int(datetime.strptime(most_recent_alert_created_date, SECURITYSCORECARD_DATE_FORMAT).timestamp())  # type: ignore
```

Comment:
Why don't we just do `arg_to_datetime` function here?
```suggestion
        most_recent_alert_timestamp = arg_to_datetime(most_recent_alert_created_date).timestamp())
```

Reply:
I'm not using `arg_to_datetime` here because the alert date returned from the API is different than the one used by XSOAR:

```python
SECURITYSCORECARD_DATE_FORMAT = "%Y-%m-%dT%H:%M:%S.%fZ"
DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
```

`arg_to_datetime` doesn't have an argument for format so I could not use it.